### DISCUSSION TITLE
Markdown to api reference

### DISCUSSION THREAD

**Discussion by arthurvanl** [scalar/scalar#1342] [Ideas]
![image](https://github.com/scalar/scalar/assets/20114845/52f5ba68-8a79-439f-ac02-19d9100cf522)

Would be amazing if we can add tags without routes attached to it so we can add information about the api

**Comment by marclave** [comment]
hi @arthurvanl ! I wonder if we should do something outside of tags for this, like a markdown section or extension?

wdyt @cameronrohani @hanspagel

**Reply by hanspagel** [reply]
I think you could use `info.description` for that purpose: https://sandbox.scalar.com/e/h-Z6d
